Framework/DataHandling/src/LoadNexusProcessed.cpp

```diff
@@ -113,7 +113,7 @@ SpectraInfo extractMappingInfo(NXEntry &mtd_entry, Logger &logger) {
   try {
     NXInt spectra_block = detgroup.openNXInt("spectra");
     spectra_block.load();
-    spectraInfo.spectraNumbers = spectra_block.sharedBuffer();
```

Comment:
Do we have a similar issue with copying to the event nexus case? The `vecBuffer()` return will now be copied into spectraNumbers rather than just copying a pointer.

Reply:
`vecBuffer` returns a reference to the container so I think this should be okay?